RELEASE_PROCESS.md

```diff
@@ -0,0 +1,151 @@
+
+Most PRs shouldn't have any Expeditor control labels. The patchlevel (4.18.X) will be automatically incremented by the [expeditor configuration](https://github.com/inspec/inspec/blob/44fe144732e1e0abb2594957a880c5f1821e7774/.expeditor/config.yml#L117) and the [version bump script](https://github.com/inspec/inspec/blob/master/.expeditor/update_version.sh).
+
+ * Expeditor: Bump Minor Version - Use when a significant new feature is being released.
+ * Expeditor: Skip Changelog - Should only be used for dummy PRs.
+ * Expeditor: Skip Version Bump - Use for non-code-change PRs, such as website or CI changes.
+ * Expeditor: Skip Omnibus - Use for non-code-change PRs, such as website or CI changes.
+ * Expeditor: Skip Habitat - Use for non-code-change PRs, such as website or CI changes.
+
+### Click Merge
+
+This should be straightforward, assuming the big merge button is green.
+
+If the merge button is red, take a moment and reminisce about various regrets in your life, then make your choice. Merging a build that fails tests is counterproductive; we have further tests downstream during the post-merge process.
+
+You'll see a message in #inspec-notify from Expeditor that it "performed actions for merged_inspec/inspec#1234" (for PR 1234, for example). Among the messages will be links to the Omnibus and Habitat builds.
+
+### Watch Omnibus Build
+
+The Omnibus build creates operating-system-specific packages for each platform for which we release InSpec. Its [expeditor configuration](https://github.com/inspec/inspec/blob/44fe144732e1e0abb2594957a880c5f1821e7774/.expeditor/config.yml#L133) drives a [BuildKite configuration](https://github.com/inspec/inspec/blob/master/.expeditor/release.omnibus.yml) which lists exactly which platforms to build.
+
+The Omnibus build is generally reliable, if somewhat slow.
+
+When the omnibus build succeeds, omnitruck delivers the packages to various package repos in `unstable` channels for public consumption.
+
+### Watch Habitat Build
+
+The Habitat build creates Habitat .hart packages for Linux and Windows. The [Expeditor configuration](https://github.com/inspec/inspec/blob/44fe144732e1e0abb2594957a880c5f1821e7774/.expeditor/config.yml#L138) drives a [BuildKite configuration](https://github.com/inspec/inspec/blob/master/.expeditor/build.habitat.yml).
+
+The habitat build is unreliable, usually due to network timeouts on the Windows machines. It often requires manual retries by clicking the retry button in BuildKite (obtain the link from the message posted by Expeditor in #inspec-notify). If you do not retry, later steps will fail.
+
+When the hab build succeeds, the packages will be placed on the Hab builder in the `unstable` channel for public consumption.
+
+### Docker Image Built and Released
+
+We also release a Docker image, which contains an Alpine system and InSpec installed from a gem, with the EXEC of the Docker image being `inspec`. It's a simple way to ship the dependencies of inspec.
```

Comment:
Would it be correct to point to https://github.com/inspec/inspec/blob/44fe144732e1e0abb2594957a880c5f1821e7774/.expeditor/config.yml#L150 at this line, to show where the docker build is configured to happen automatically?